tokens/properties/components/TextArea.json

```diff
@@ -0,0 +1,118 @@
+        },
+        "border-width": {
+            "value": "{border.width.thin.value}"
+        },
+        "border-radius": {
+            "value": "{border.radius.base.value}"
+        },
+        "background-color": {
+            "value": "{background.color.base.default.value}"
+        },
+        "font-family": {
+            "value": "{font.family.style.body.value}"
+        },
+        "font-size": {
+            "value": "{font.size.style.body.value}"
+        },
+        "font-weight": {
+            "value": "{font.weight.style.body.value}"
+        },
+        "line-height": {
+            "value": "{font.line-height.style.body.value}"
+        },
+        "color": {
+            "value": "{font.color.base.value}"
+        },
+        "padding-inline": {
+            "value": "{dimension.spacing.small.value}"
+        },
+        "padding-block": {
+            "value": "{dimension.padding.vertical.medium.value}",
+            "comment": "to be replaced by spacing token in the future"
+        },
+        "placeholder": {
+            "font-family": {
+                "value": "{font.family.style.placeholder.value}"
+            },
+            "font-size": {
+                "value": "{font.size.style.placeholder.value}"
+            },
+            "font-weight": {
+                "value": "{font.weight.style.placeholder.value}"
+            },
+            "line-height": {
+                "value": "{font.line-height.style.placeholder.value}"
+            },
+            "color": {
+                "value": "{font.color.subtle.value}"
+            }
+        },
+        "disabled": {
+            "color": {
+                "value": "{font.color.disabled.value}"
+            },
+            "border-color": {
+                "value": "{border.color.base.disabled.value}"
+            },
+            "background-color": {
+                "value": "{background.color.field.disabled.value}"
+            }
+        },
+        "hover": {
+            "border-color": {
+                "value": "{border.color.base.hover.value}"
+            }
+        },
+        "active": {
+            "border-color": {
+                "value": "{border.color.base.focus.value}"
+            },
+            "box-shadow": {
+                "value": "{box-shadow.inset.focus.progressive.value}"
+            }
+        },
+        "error": {
+            "border-color": {
+                "value": "{border.color.feedback.error.value}"
+            },
+            "active": {
+                "box-shadow": {
+                    "value": "{box-shadow.inset.focus.error.value}"
+                }
+            }
+        },
+        "warning": {
+            "border-color": {
+                "value": "{border.color.feedback.warning-field.value}"
+            },
+            "active": {
+                "box-shadow": {
+                    "value": "{box-shadow.inset.focus.warning.value}"
+                }
+            }
+        },
+        "readOnly": {
```

Comment:
Please note that read-only fields present a different background color and limited interaction (user cannot modify the value). Still, they can be activated (focused) and hovered, and thus they should display the border styles corresponding to those states.